### ISSUE TITLE
Why does classification demo have 2 units in final Dense layer?

### ISSUE THREAD

**Issue by tinkermachine** [CyberZHG/keras-bert#95]
I thought the IMDB data set would simply have one label (a 0 for neg or a 1 for pos sentiment). So why do we have 2 units in the final dense layer? I don't believe we are one-hot encoding the labels...

`outputs = keras.layers.Dense(units=2, activation='softmax')(dense)`

**Comment by tinkermachine** [comment]
This is resolved. I misunderstood the theory behind single label classification with integer labels.

**Comment by Xiaoping777** [comment]
I have the same question here, why not use "sigmoid" function for the activation function, with the units=1?
